### ISSUE TITLE
"Previous appeals" should show appeals from UTRS 1 era

### ISSUE THREAD

**Issue by supertassu** [UTRS2/utrs#216]
split out from #214

**Comment by dqwiki** [comment]
I think this has been dealt with no?

**Comment by JJMC89** [comment]
> I think this has been dealt with no?

I don't think so. Going to appeal #31060 doesn't show any of the UTRS 1 appeals for that user in the previous appeals section.

**Comment by dqwiki** [comment]
UTRS 1 appeals are essentially of no use now, so closing
